src/main/java/frc/team4909/robot/subsystems/intake/CargoIntakeIn.java

```diff
@@ -15,6 +15,6 @@ protected void execute() {
 
     @Override
     protected boolean isFinished() {
-        return Robot.intakeSubsystem.getCargoIntakeCurrent() > RobotConstants.cargoIntakeCurrentLimit;
+        return false;
```

Comment:
Never stop? What stops it? And if it is intended to not be stopped here, should it be an InstanceCommand rather than a Command?

Reply:
we're still trying to correct this logic, as it's not working as intended– this was also part of last minute testing